### ISSUE TITLE
Title Page Top Margin

### ISSUE THREAD

**Issue by PeterLiPeide** [amunn/msu-thesis#10]
Hi, thank you so much for the template. I recently submit my PhD thesis to the graduate school. They pointed out that the top margin of my title page is 2.5 inches and should be 2 inches. I know you commented this issue in the template and on one knows how they measure the margin. I change the \setlength{\droptitle}{.25in} to \setlength{\droptitle}{-.25in} to move my title up for 0.5 inch. I am going to submit the revise and will share the result once I hear back from the graduate school.

**Comment by amunn** [comment]
@PeterLiPeide  Thanks for the report. The test document correctly has the title 2in from the top, so I suspect this is a problem with your document and not the class. The office sometimes measures to the top of a line an sometimes to the bottom of a line, but this is not 0.5in :) If you want to email me your thesis I can take a quick look, but your solution will likely work for your particular document. 
![Screen Shot 2021-06-30 at 10 03 10 PM](https://user-images.githubusercontent.com/633053/124054210-8f542680-d9ef-11eb-9f9e-9ddd569968ad.png)

**Comment by PeterLiPeide** [comment]
@amunn Thank you for the reply. I just realized and solved the issue. I think \droptitle command assumes the top margin to be the gap between the paper top edge and the first line of the title. However, the graduate school defines the margin as the gap between center of the title and the paper top edge. Depending on how many lines your title has, the command **\setlength{\droptitle}{.25in}** should be changed accordingly. I tried three different values in the command, printed title pages out and measured the margin with a ruler. For me, the value should be 0in as I have two lines for the title. It would be great if this can be pointed out in the template so that users can realize and make changes by themselves. Thank you again for providing this wonderful template.
